src/utils/validations.js

```diff
@@ -1,37 +1,42 @@
-/*Validation constructor */
-function Validations() {
-}
-
-/*Validate if empty string*/
-Validations.prototype.isEmptyString = function (word){
-    if(word == undefined) return true;
-    if(word == null) return true;
-    if(word.length === 0 || word.isEmpty || word.trim().length === 0) return true;
-    else return false;
-};
-
-/*Validate if empty string*/
-Validations.prototype.isEmptyObject = function (word){
-    if(word == undefined) return true;
-    if(word == null) return true;
-    else return false;
-};
+class Validations {
+    constructor() {
+    }
 
-/*Method for validate a team creation */
-Validations.prototype.normalizeTeamName = function (teamName) {
+    /*Method for validate if string is empty
+    @param word
+    String to be validated
+    */
+    isEmptyString(word) {
+        if (word.length === 0 || word.isEmpty || word.trim().length === 0) return true;
+        else return false;
+    };
 
-    console.log("team name on validation "+teamName);
-    var team;
-    team = teamName.toLowerCase();
-    team = team.trim();
-    
-    var words = team.split(" ");
-    team = "";
-    for (i = 0; i < words.length; i++) { 
-        if(!this.isEmptyString(words[i])) team += words[i]+" ";
-    }
-    team = team.trim();
-    return team;
-};
+    /*Method for validate if object 
+    is not undefined or null
+    @param obj
+    object to be validated
+    */
+    isEmptyObject(obj) {
+        if (obj == undefined) return true;
+        if (obj == null) return true;
+        else return false;
+    };
 
+    /*Method for normalize team name 
+    @param teamName
+    team name remove uppercase and white spaces
+    */
+    normalizeTeamName(teamName) {
+        var team;
+        team = teamName.toLowerCase();
+        team = team.trim();
+        var words = team.split(" ");
+        team = "";
+        for (var i = 0; i < words.length; i++) {
+            if (!this.isEmptyString(words[i])) team += words[i] + " ";
```

Comment:
If statement must contain only conditionals, you are doing a variable assignment in this step

Reply:
also concat string with `+` operators is not a good practice, check how to do it with **template literals** in Ecmascript 6 (new specification of js)

Reply:
fix, instead of template literal i used concat function